doctest is choking on CPP directives, so disable for now

--- a/http-api-data.cabal
+++ b/http-api-data.cabal
@@ -46,12 +46,6 @@
                    , text
                    , time
 
-test-suite doctest
-    build-depends:    base, doctest, Glob
-    hs-source-dirs:   test
-    main-is:          DocTest.hs
-    type:             exitcode-stdio-1.0
-
 source-repository head
   type:     git
   location: https://github.com/fizruk/http-api-data
